go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	golang.org/x/tools v0.0.0-20200616133436-c1934b75d054
 	google.golang.org/grpc v1.31.1
 	google.golang.org/protobuf v1.24.0
+	gotest.tools/gotestsum v0.4.2 // indirect
```

Comment:
Wondering why this is added, as this is not being used anywhere. Can you do `go mod tidy` for the go tool to clean it up?